packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -35,10 +37,32 @@ const StyledSiteBody = styled.div<StyledSiteBodyProps>`
   }
 `;
 
+const BLMAlert: React.FC = () => (
+  <Box
+    css={{backgroundColor: '#000', color: '#fff', borderColor: '#000'}}
+    borderBottomWidth="borderWidth20"
+    borderBottomStyle="solid"
+    paddingLeft="space70"
+    paddingRight="space70"
+    paddingTop="space40"
+    paddingBottom="space30"
+    role="status"
+    display="flex"
+    justifyContent="center"
+  >
+    <strong>Black Lives Matter</strong>
+    &nbsp;&nbsp;
+    <Anchor css={{color: '#fff'}} href="https://twitter.com/twilio/status/1266444750065934337">
```

Comment:
can this use a token value instead of CSS? Also, can the hover text be white without the underline?

Reply:
Also focus state. Both are inaccessible presently

Reply:
Hoping we can use <Text as="a"> for this?

Reply:
there's no token for the #000, but I swapped #fff to a token